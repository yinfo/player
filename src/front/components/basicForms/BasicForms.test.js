// @flow

import React from 'react';
import { shallow } from 'enzyme';
import BasicForms from './BasicForms';

describe('BasicForms component', () => {
  it('renders as expected', () => {
    const props = {
      headerTitle: 'basic form header title',
    };

    const component = shallow(<BasicForms {...props} />);
    expect(component).toMatchSnapshot();
  });
});
