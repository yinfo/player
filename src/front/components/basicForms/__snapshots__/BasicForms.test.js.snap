// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicForms component renders as expected 1`] = `ShallowWrapper {}`;
